tests/plugins1/_batch_test/test/test.mustache

```diff
@@ -41,9 +41,7 @@ for NODE in $(seq 1 1 "$PSIJ_TEST_BATCH_EXEC_COUNT"); do
 done
 
 export PSIJ_NODEFILE
+{{#job.spec.inherit_environment}}{{/job.spec.inherit_environment}}{{^job.spec.inherit_environment}}env --ignore-environment \{{/job.spec.inherit_environment}}{{#env}}
+export {{name}}="{{value}}" {{/env}}
 
-{{#job.spec.inherit_environment}}env \{{/job.spec.inherit_environment}}{{^job.spec.inherit_environment}}env --ignore-environment \{{/job.spec.inherit_environment}}{{#env}}
-{{name}}="{{value}}" \
-{{/env}}{{#psij.launch_command}}{{.}} {{/psij.launch_command}}
-
-echo "$?" > "{{psij.script_dir}}/$PSIJ_BATCH_TEST_JOB_ID.ec"
\ No newline at end of file
+echo "$?" > "{{psij.script_dir}}/$PSIJ_BATCH_TEST_JOB_ID.ec"
```

Comment:
I' m trying to understand this change. It was:

```bash
export --ignore-environment \
    A='a' \
    b='b' \
    command
```

That is, `env` is used to set the variables and run the command.

It looks like now it will expand to
```bash
export --ignore-environment \
    export A='a'
    export B='b'
    command
```

Here `env` is used to set the first variable, although I think this should fail since env will try to run the `export` executable, which is a bash builtin, so it will fail. Even if not, it will only set the first variable. Then the second and subsequent variables will be set, and the command run with the inherited environment (since the scope of the effects of  `env --ignore-environment` has ended) and all but the first variable set.

Reply:
jeah, my 'fix' was not a fix apparently.  The problem is that I don't get the 

```sh
export --ignore-environment \
    A='a' \
    b='b' \
    command
```

but rather see 
```sh
export --ignore-environment \
    A='a' \
    
    b='b' \
    command
```

(See [mustache.txt](https://github.com/ExaWorks/psij-python/files/14311838/mustache.txt) for example).  I tried to get rid of the newline, but the escaping is a bit funny with mustache :-)  Reverted now, please let me know if you have any advise.

Reply:
Indeed, and we weren't testing for multiple env vars.

I think the issue is that there is a newline after `{{#env}}`, which puts a newline before each env var line, and there is one after each env var, which adds another.

The following seems to work for me:
```
{{#job.spec.inherit_environment}}env \{{/job.spec.inherit_environment}}{{^job.spec.inherit_environment}}env --ignore-environment \{{/job.spec.inherit_environment}}{{#env}}
{{name}}="{{value}}" \{{/env}}
{{#psij.launch_command}}{{.}} {{/psij.launch_command}}
```

I.e., only add newline before each env var and before the launch command.

Reply:
Perhaps it would be wise to fix in all executors.

Reply:
I don't think the other executors suffer from the same problem as they create individual `export` directives per env pair?